org.eclipse.scanning.test/src/org/eclipse/scanning/test/malcolm/device/LoopingMockedMalcolmDevice.java

```diff
@@ -20,6 +20,9 @@
 import org.eclipse.scanning.api.scan.ScanningException;
 
 /**
+ *
+ * TODO DAQ-1004, see comment in {@link MockedMalcolmDevice}
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)